scripts/install_runtime_dependencies.sh

```diff
@@ -1,23 +1,25 @@
 #!/bin/bash
 
-BUILD_DEPENDS=(gnupg2 gpg)
+BUILD_DEPENDS=(gpg)
```

Comment:
Are you sure that we don't need the `gnupg2` package in any possible case? It is ok if not, but I added it from the Darktable installation guide

Reply:
Debian requires only with `gpg` to add keys. `gnupg2` is a dummy transitional package, pointing to `gnupg`, and `gpg` is one dependency of `gnupg`. `gpg` is the essential package. If it's not sufficient, it should be `gnupg` here.